controllers/articles.controller.js

```diff
@@ -11,3 +11,12 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  getAllArticles()
+    .then((articles) => res.status(200).send({ articles }))
```

Comment:
Great choice of key to send the data back on, this is nice and descriptive for another dev using your api 👍